src/panel/category/PoiItemListPlaceholder.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ItemList, Item } from 'src/components/ui/ItemList';
+import PlaceholderText from 'src/components/ui/PlaceholderText';
+
+const PoiItemPlaceholder = () =>
+  <div className="category__panel__item">
+    <div className="poiTitleImage u-placeholder" />
+    <PlaceholderText length={25} tagName="h3" className="category__panel__name"/>
+    <PlaceholderText length={15} tagName="p" className="category__panel__type" />
+    <PlaceholderText length={25} tagName="p" className="category__panel__address" />
+    <div className="openingHour"><PlaceholderText length={15} /></div>
+  </div>;
+
+const PoiItemListPlaceholder = ({ nbItems = 6 }) =>
+  <ItemList className="category__panel__items category__panel__items--placeholder">
+    {Array.from({ length: nbItems }).map((_item, index ) => <Item key={index}>
```

Comment:
Isn't "Array(nbItems)" simpler?

Reply:
`Array(nbItems)` is a strange thing and is not clearly defined in the JS API. Here it wouldn't work as it doesn't return a real array but "something" with 5 slots…
As a result, the `.map( )` function doesn't transform the items and the resulting stuff is not what we want.

You can try in a browser console.

Firefox considers it's an array filled with `undefined` 
![Capture d’écran de 2020-02-24 15-12-45](https://user-images.githubusercontent.com/243653/75159952-7c558600-5719-11ea-8334-39f99dc46930.png)

and Chrome tells it's `empty`
![Capture d’écran de 2020-02-24 15-24-10](https://user-images.githubusercontent.com/243653/75160098-c3437b80-5719-11ea-8b22-f84f36a6d7cd.png)

Reply:
ah yes, ughh JS

Reply:
In many projects it's common to use util functions to do that instead of using `Array.from`. For example, if we used the Ramda util library https://ramdajs.com/docs/#repeat.
If we do that often maybe it will be nice to implement a common util function.